seqr/utils/search/elasticsearch/es_search.py

```diff
@@ -1413,7 +1413,8 @@ def _location_filter(genes, intervals, exclude_locations):
                     }
                 } for key in ['xpos', 'xstop']]
                 interval_q = _build_or_filter('range', range_filters)
-                interval_q |= Q('range', xpos={'lte': xstart}) & Q('range', xstop={'gte': xstop})
+                interval_q |= Q('range', xpos={'lte': xstart}) & Q('range', xstop={'gte': xstop}) &\
+                              Q('script', script="doc['xpos'].value / 1000000000 == doc['xstop'].value / 1000000000")
```

Comment:
script queries are really inefficient, so in general we don't want to use them. I think a better approach would be jusy to add additional checks here that `xstart` and `xstop` are both on the  same chromosome as `interval['chrom']`. You can do this by just adding the following to the current query:
```suggestion
                interval_q |= Q('range', xpos={'lte': xstart}) & Q('range', xstop={'gte': xstop}) &
                              Q('range', xpos={'gte': get_xpos(interval['chrom'], 0)}) & Q('range', xstop={'lte': get_xpos(interval['chrom'], MAX_POS)})
```

Reply:
Sounds good. BTW. Do we need to handle the case that the `end` is on a different chromosome and the interval locates after the `xstart` on the same chromosome? Or the interval is before the `xstop` on the same chromosome of `xstop`?

Reply:
No we don't need any special handling for that. If this is a breakend event and either xpos or xstop falls directly inside a filtered interval, then the `range_filters` above will capture that and those variants will be returned, which is the desired behavior. And otherwise if they are on different chromosomes we do not want them returned, and this change will handle that